README.md

```diff
@@ -37,6 +37,15 @@ cat > /etc/pcw.ini << EOT
 url = https://publiccloud.your.vault.server/vault
 user = Your_VAULT_USER
 password = VAULT_USER_PASSWORD
+namespaces = VAULT_NAMESPACE, VAULT_NAMESPACE_2, ...
+
+[vault.namespace.XXX]
+# XXX is the name of a namespace given in vault.namespaces
+# provider should be ec2, azure or csp
+providers = csp1[, csp2]...
```

Comment:
why we need this ? isn't it easier just apply namespace for all providers ?

Reply:
This is the default => and this hint is missing in the here, will do it
see: https://github.com/cfconrad/pcw/pull/34/files#diff-0ad2fc8593cc924b952fd5277e6a11d7R76

We need this, cause currently, not all credentials are well configured. For instance I think `qam` has only configured *ec2*. @pdostal plz correct me if I'm wrong.
So with out it, we will never get successful update.

Reply:
That is true. We still don't have any other provider than EC2 available.

Reply:
let's leave this to the end , in case patch would be simplified in a way I see it , this will not be an issue that you don't have all configured